src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4638,4 +4638,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ConstantStringTooLong" xml:space="preserve">
     <value>Length of String constant exceeds current memory limit.  Try splitting the string into multiple constants.</value>
   </data>
+  <data name="ERR_LoadDirectiveOnlyAllowedInScripts" xml:space="preserve">
+    <value>#load is only allowed in scripts</value>
```

Comment:
The intent is that this error will only show up in "regular" C# code (.cs files).